R/4_standard_error.R

```diff
@@ -135,6 +157,77 @@ standard_error.default <- function(model, method = NULL, verbose = TRUE, ...) {
 }
 
 
+.get_vcov <- function(x,
+                      vcov_fun = "vcovHC",
+                      vcov_args = NULL,
+                      component = "conditional",
+                      method = "any",
+                      ...) {
+
+  dots <- list(...)
+
+  if (is.null(vcov_args)) {
+    vcov_args <- list()
+  }
+
+  # deprecated: `vcov_estimation`
+  if (is.null(vcov_fun) && "vcov_estimation" %in% names(dots)) {
+    vcov_fun <- dots[["vcov_estimation"]]
+  }
+
+  # deprecated: `robust`
+  if (isTRUE(dots[["robust"]]) && is.null(vcov_fun)) {
+    dots[["robust"]] <- NULL
```

Comment:
Do we want a warning?

Reply:
I will include a warning in `.get_vcov()` since everything flows through there.